qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -80,6 +80,24 @@ class BasisTranslator(TransformationPass):
         translating a 2 qubit operation on qubit 0 and 1 that the output might
         have ``u`` on qubit 1 and ``x`` on qubit 0. Typically running this pass
         a second time will correct these issues.
+
+    .. translation_errors:
```

Comment:
I think you need a leading underscore on this link target?

Reply:
Sigh, yeah I missed that. I should have run a docs build locally before pushing :)